microprofile/tests/tck/tck-cdi-lang-model/pom.xml

```diff
@@ -75,8 +75,8 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.glassfish</groupId>
-            <artifactId>jakarta.el</artifactId>
+            <groupId>org.glassfish.expressly</groupId>
```

Comment:
Is it OK to change a TCK `pom.xml`?

Reply:
As far as I can tell the TCK depends only on the el-api (as it should), and it is up to the implementation being tested to provide the implementation of el.

Reply:
OK!